lxd/storage/drivers/driver_ceph_volumes.go

```diff
@@ -418,33 +424,35 @@ func (d *ceph) CreateVolumeFromCopy(vol Volume, srcVol Volume, copySnapshots boo
 				if srcVol.IsSnapshot() {
 					srcParentName, srcSnapOnlyName, _ := api.GetParentAndSnapshotName(srcVol.name)
 					snapshotName = fmt.Sprintf("snapshot_%s", srcSnapOnlyName)
-					parentVol = NewVolume(d, d.name, srcVol.volType, srcVol.contentType, srcParentName, nil, nil)
+					parentVol = VolumeCopy{
```

Comment:
Should we use NewVolumeCopy here?

Reply:
Thanks, that was a leftover, I have introduced `NewVolumeCopy()` later and thought I have caught all occurrences.